lib/chelsea/cli.rb

```diff
@@ -39,15 +39,21 @@ def process!
         _set_config # move to init
       elsif @opts.clear?
         require_relative 'db'
-        Chelsea::DB.new().clear_cache
+        Chelsea::DB.new.clear_cache
         puts "OSS Index cache cleared"
       elsif @opts.file? && @opts.iq?
         dependencies = _process_file_iq
         _submit_sbom(dependencies)
+      elsif !@opts.file? && @opts.iq?
+        puts "Missing the --file argument. It is required with the --iq argument."
+        exit 1
       elsif @opts.file?
         _process_file
       elsif @opts.help? # quit on opts.help earlier
         puts _cli_flags # this doesn't exist
+      else
+        puts "Missing arguments! Chelsea did nothing. Try providing the --file <Gemfile.lock> argument."
```

Comment:
https://ruby-doc.org/core-2.4.1/Kernel.html#method-i-abort

If you are gonna put and exit with an error, abort might be your cup o tea :)

Also writes to stderr, which is probably good if someone wanted to redirect errors.

Reply:
Yummy! Thanks! Done and done: 93258f3